doc/02-installation.md.d/For-Container.md

```diff
@@ -0,0 +1,179 @@
+# Installing Icinga 2 in Containers
+
+To be able to run Icinga 2 in a containerized environment, you'll need to set up a few things.
+This guide will help you get started with running Icinga 2 in a container using our official images.
+
+## Prerequisites
+
+- A container runtime such as [Docker](https://www.docker.com) or [Podman](https://podman.io) installed on your system.
+- Basic knowledge of how to use Docker or Podman commands.
+- A basic understanding of Icinga 2 and its configuration.
+
+## Getting Started
+
+First, create a dedicated docker network for Icinga 2 to ensure that all containers can communicate with each other:
+
+```
+docker network create icinga
+```
+
+Next, start an Icinga 2 master container using the official Icinga 2 image. By default, all Icinga 2 containers will
+listen on port `5665` from within the docker network, but you can expose this port on a different port on your host
+system if needed. The following command will start an Icinga 2 master container with the necessary configurations:
+
+```
+docker run --rm --detach \
```

Comment:
Should this (and other commands in this file) really pass `--rm`? I mean the data is safe inside the volume, but deleting the container when is stops still doesn't sound right.